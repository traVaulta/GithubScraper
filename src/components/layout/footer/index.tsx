import React from 'react';

export const AppFooter = () => (
    <footer className="app-footer app-footer--outline">
        <div>
            &copy;GithubScraper, Croatia, 2020
        </div>
        <div className="app-footer app-footer--contact">
            <a href="mailto:matijacvrk@gmail.com">matijacvrk@gmail.com</a>
        </div>
    </footer>
);
