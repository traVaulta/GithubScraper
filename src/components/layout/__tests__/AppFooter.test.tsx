import React from 'react';
import {render} from '@testing-library/react';

import {AppFooter} from '../footer';

describe(`${AppFooter.name}`, () => {
    it('should render', () => {
        const {asFragment} = render(<AppFooter/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
