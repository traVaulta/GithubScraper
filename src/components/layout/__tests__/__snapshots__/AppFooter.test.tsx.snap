// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should render 1`] = `
<DocumentFragment>
  <footer
    class="app-footer app-footer--outline"
  >
    <div>
      Â©GithubScraper, Croatia, 2020
    </div>
    <div
      class="app-footer app-footer--contact"
    >
      <a
        href="mailto:matijacvrk@gmail.com"
      >
        matijacvrk@gmail.com
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
